inc/saibfd.h

```diff
@@ -88,6 +88,11 @@ typedef enum _sai_bfd_encapsulation_type_t
      */
     SAI_BFD_ENCAPSULATION_TYPE_NONE,
 
+    /**
+     * @brief SRV6 Tunnel Encapsulation | L2 Ethernet header | SRV6 Header | IP header | Original BFD packet
```

Comment:
Shouldn't the IP (ipv6) header come before SRV6 header in the comment?

Reply:
Updated